pkgs/by-name/el/elia/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+let
+  textual = python3Packages.textual.overrideAttrs rec {
+    version = "0.79.1";
+    src = fetchFromGitHub {
+      owner = "Textualize";
+      repo = "textual";
+      rev = "v${version}";
+      hash = "sha256-laGhjlsFWcSgDnTjLSJMdfM6BRZTezIzFSsyVUhpr3Q=";
+    };
+  };
```

```suggestion
  textual = python3Packages.textual.overridePythonAttrs rec {
    version = "0.79.1";
    src = fetchFromGitHub {
      owner = "Textualize";
      repo = "textual";
      tag = "v${version}";
      hash = "sha256-laGhjlsFWcSgDnTjLSJMdfM6BRZTezIzFSsyVUhpr3Q=";
    };
    
    # Fails on aarch64-linux
    # RuntimeError: There is no current event loop in thread 'MainThread'.
    disabledTests = [ "test_wrap_around_selector" ];
  };
```
Also note the change to `tag`. See https://github.com/NixOS/nixpkgs/pull/355973

---

Hmm, using overridePythonAttrs seems to change the version of the derivation for textual. I had wondered why the version never changed.

---

However when I disable that test, upon building the tests fail. 
Tail of the failure log:
```
v^*'
            ) from e
E           AttributeError: 'module' object at textual.document._syntax_aware_document has no attribute 'get_language'

/nix/store/ziygfgrrfki93xcr567isy6j2431qp7i-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/monkeypatch.py:92: AttributeError
=========================== short test summary info ============================
FAILED tests/test_features.py::test_textual_env_var - AssertionError: assert None is not None
FAILED tests/text_area/test_languages.py::test_register_language - ModuleNotFoundError: No module named 'tree_sitter_languages'
FAILED tests/text_area/test_languages.py::test_register_language_existing_language - AssertionError: assert 0 > 0
FAILED tests/text_area/test_languages.py::test_language_binary_missing - AttributeError: 'module' object at textual.document._syntax_aware_document ...
============ 4 failed, 2624 passed, 1 skipped, 3 xfailed in 34.66s =============
```

---

You should be able to disable these as well. I'm guessing the errors are from now resolved upstream issues -- but it'd also be good to check their issues just in case